doc/reference/mobile.rst

```diff
@@ -0,0 +1,204 @@
+:banner: banners/mobile.jpg
+:types: api
+
+.. _reference/mobile:
+
+==================
+Mobile JavaScript
+==================
+
+Introduction
+============
+
+In Odoo 10.0 We released new mobile app, allows you to access all **Odoo apps** (Even your customized modules). 
+
+The application is combination of **Odoo Web** and **Native Mobile components**, In other words it is a Odoo Web instance loaded inside native WebView container of mobile.
+
+This page documents how you can access mobile native components like Camera, Vibration, Notification, Toast through Odoo Web (via JavaScript). For this, you no need to be a mobile developer, if you know Odoo JavaScript API you can access all available mobile features.
+
+.. warning:: This features works with **Odoo Enterprise 10.0+** only
+
+How it works? 
+=============
+
+Internal working of Mobile application:
+
+.. image:: images/mobile_working.jpg
+    :align: left
+
+Ofcourse, Its web page that loads on Mobile Native Web container. But it is integrated in a such way so you can access native resources from your web JavaScript.
+
+WebPages (Odoo Web) is on the top of each layer, where second layer is a Bridge between Odoo Web (JS) and Native component of mobile.
+
+When any call from JavaScript triggered it passes through Bridge and Bridge pass it to native invoker to perform that action.
+
+When native component done their work, It again passed to Bridge and you get the output in JavaScript.
+
+Process time taken by Native component is depend on what you are requesting from Native resources. Like Camera or GPS Location.
+
+How to use?
+===========
+
+As Odoo Web Framework, Mobile api can be used anywhere by getting object of **web_mobile.rpc**
+
+.. image:: images/odoo_mobile_api.png
+    :align: left
+
+Mobile rpc object provide list of methods available (Works only with Mobile App). 
+
+Just check for method availability and execute it.
+
+Methods
+-------
+
+.. note:: Each of the methods returns JQuery Deffered object with returning data JSON dictionary
+
+``showToast(args)``
```

Comment:
Use ``.. js::function:` sphinx directive to document JS functions: http://www.sphinx-doc.org/en/stable/domains.html#the-javascript-domain